mono/metadata/sre.c

```diff
@@ -2803,7 +2803,8 @@ reflection_methodbuilder_to_mono_method (MonoClass *klass,
 	m->slot = -1;
 	m->flags = rmb->attrs;
 	m->iflags = rmb->iattrs;
-	m->name = mono_string_to_utf8_image_ignore (image, rmb->name);
+	m->name = mono_string_to_utf8_image (image, rmb->name, error);
```

Comment:
If ignoring the error was required, or we want to be conservative, then I can here introduce a local error and ignore it. It isn't ideal, but it is better than pushing through the error-ignoring down multiple layers as was previously done.